plugin/callgraph/callgraph.go

```diff
@@ -103,21 +118,22 @@ func newCallGraph(fileName string, rootNode *sitter.Node, imports []*ast.ImportN
 	for _, wildcardImport := range wildcardImports {
 		// For wildcard imports, we add a call to importeditem//*
 		// assuming that anything under that namespace is posssibly used
-		cg.AddEdge(fileName, rootNode, wildcardImport.NamespaceTreeNode, wildcardImport.Namespace, wildcardImport.NamespaceTreeNode)
+		cg.AddEdge(fileName, rootNode, wildcardImport.NamespaceTreeNode, wildcardImport.Namespace, wildcardImport.NamespaceTreeNode, []CallArgument{})
```

Comment:
`AddEdge` seems like exported. Over time we should not break exported APIs. Is this required required to be exported or can this be internal to call graph builder - `addEdge(...)`?

May be review the methods and make everything private except for the bare minimum. This way the API will be more stable from a consumer perspective. Better to start small and then open up access to stable APIs.

Remember: We are building a framework that will be used by developers

Reply:
makes sense, did misc refactors